frontend/packages/react-app/src/components/organisms/CampaignDetail/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Card, Flex, Box, Text } from "rimble-ui";
+import { CampaignInfo } from "../../../interfaces";
+
+export interface CampaignDetailProps {
+  readonly campaignInfo: CampaignInfo;
+}
+
+const CampaignDetail = ({ campaignInfo }: CampaignDetailProps) => {
+  return (
+    <Card mt={2}>
+      <Box>
+        <Flex style={{ alignItems: "center", justifyContent: "left" }}>
+          <Box>
+            <Text fontSize={2} color="gray">
+              Campaign Name
+            </Text>
+            <Text fontSize={3} fontWeight="bold">
+              {campaignInfo.campaignMetadata.name}
+            </Text>
+          </Box>
+        </Flex>
+        <Flex mt={4} style={{ alignItems: "center", justifyContent: "left" }}>
+          <Box>
+            <Text fontSize={2} color="gray">
+              Deposited tokens amount
+            </Text>
+            <Text fontSize={3} fontWeight="bold">
+              {campaignInfo.depositAmount}
+            </Text>
+          </Box>
```

Comment:
It seems can be extracted such as `Item` to Unify item appearance.

Reply:
Fixed